src/Java2Lesson1/Cat.java

```diff
@@ -0,0 +1,41 @@
+package Java2Lesson1;
+
+public class Cat implements JumpAndRun {
+
+    int maxHeight;
+    int maxLength;
+    String name;
+
+    public Cat (int maxHeight, int maxLength, String name) {
```

Comment:
отступ между именем метода и параметрами не нужно делать
```suggestion
    public Cat(int maxHeight, int maxLength, String name) {
```